### PR TITLE
Adding debug property to accessory to lessen the chatter

### PR DESCRIPTION
**Pull request by smith288**
Adding debug property to accessory to lessen the chatter in the homebridge log.

### CODE REVIEW

**Comment by apexad** [comment]
Rename to `log_polling` and I will merge.

**Comment by smith288** [comment]
Done

File: README.md [hunk 1]
```diff
@@ -36,7 +37,8 @@ Field                   | Description
 **state**               | state command.  Examples: `./check_state.js` or `node state.js` (required)
 **status_update_delay** | Time to have door in opening or closing state (defaults to 15 seconds)
 **poll_state_delay**    | Time between polling for the garage door's state (leave blank to disable state polling)
-**ignore_erros**        | Causes the plugin to replace 'STOPPED' status with 'CLOSED'
+**ignore_errors**       | Causes the plugin to replace 'STOPPED' status with 'CLOSED'
+**log_puling**          | Will log every single status check to the homebridge log
```
Comment (apexad) [inline README.md]: log_polling

File: README.md [hunk 2]
```diff
@@ -20,7 +20,8 @@ Configuration sample:
     "close": "./close.sh",
     "state": "./check_state.sh",
     "status_update_delay": 15,
-    "poll_state_delay": 20
+    "poll_state_delay": 20,
+    "log_pulling": true
```
Comment (apexad) [inline README.md]: log_polling

File: index.js [hunk 3]
```diff
@@ -83,7 +84,9 @@ GarageCmdAccessory.prototype.getState = function(callback) {
       if (state === 'STOPPED' && accessory.ignoreErrors) {
         state = 'CLOSED';
       }
-      accessory.log('State of ' + accessory.name + ' is: ' + state);
+      if(accessory.log_polling)
+        accessory.log('State of ' + accessory.name + ' is: ' + state);
```
Comment (apexad) [inline index.js]: Can you please wrap this in { } even though it is only one line. Keeps the code organized.

**Comment by apexad** [comment]
Also, I would really appreciate it if you flatten all the commits(and force push) to 1 commit, with a message something like:
`Add log_polling option to stop the chatter in homebridge logs`

File: README.md (hunk 1 above)
Comment (smith288) [reply README.md]: Are you kidding.... *sigh*

**Comment by apexad** [comment]
Thanks for the PR. I will make this change, but include it as part of other minor enhancements.

**Comment by apexad** [comment]
Some additional adjustments other than the ones I mentioned were made, but this change was added in v0.3.0

**Pull request closed without merging**